### PR TITLE
Fix broken links to MyST documentation

### PR DESCRIPTION
**Pull request by ksuess**

### CODE REVIEW

**Comment by ksuess** [comment]
I'm not quite sure about the new links. Would you mind checking, @stevepiercy?

File: docs/contributing/authors.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ This renders in the HTML `<head>` section as follows.
 <meta content="Plone, Trainings, SEO, meta, presentation, exercises, solutions, spellcheck, linkcheck, lexer" name="keywords" />
 ```
 
-Additional {term}`Open Graph` metadata is implemented through the Sphinx extension [`sphinxext-opengraph`](https://github.com/wpilibsuite/sphinxext-opengraph) and the [MyST `html_meta` directive](https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#setting-html-metadata), which resolves to the [Docutils `meta` directive](https://docutils.sourceforge.io/docs/ref/rst/directives.html#metadata).
+Additional {term}`Open Graph` metadata is implemented through the Sphinx extension [`sphinxext-opengraph`](https://github.com/wpilibsuite/sphinxext-opengraph) and the [MyST `html_meta` directive](https://myst-parser.readthedocs.io/en/latest/configuration.html#setting-html-metadata), which resolves to the [Docutils `meta` directive](https://docutils.sourceforge.io/docs/ref/rst/directives.html#metadata).
```
Comment (stevepiercy) [inline docs/contributing/authors.md:50]:
```suggestion
Additional {term}`Open Graph` metadata is implemented through the Sphinx extension [`sphinxext-opengraph`](https://github.com/wpilibsuite/sphinxext-opengraph) and the [MyST `html_meta` directive](https://myst-parser.readthedocs.io/en/latest/configuration.html#global-configuration), which resolves to the [Docutils `meta` directive](https://docutils.sourceforge.io/docs/ref/rst/directives.html#metadata).
```

File: docs/contributing/writing-docs-guide.md [hunk 2]
```diff
@@ -32,15 +32,15 @@ The following are frequently used snippets and examples.
...
 **Official MyST documentation**
 
-- [The MyST Syntax Guide](https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html)
-- [MyST Syntax Reference](https://myst-parser.readthedocs.io/en/latest/syntax/reference.html)
+- [MyST authoring/typography](https://myst-parser.readthedocs.io/en/latest/syntax/typography.html)
+- [MyST Syntax tokens](https://myst-parser.readthedocs.io/en/latest/syntax/reference.html)
 ```
 
 
 #### Targets and Cross-Referencing
```
Comment (stevepiercy) [inline docs/contributing/writing-docs-guide.md:40]:
```suggestion
#### Cross-referencing
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
I found a couple more things to fix, too.

File: docs/conf.py [hunk 3]
```diff
@@ -104,7 +104,7 @@
     r"https://www.dipf.de/.*",  # a timeout from time to time
     r"https?://plone-conference.localhost.*",
     # Example domain
-    "https://domain-a.com/data.json",
+    "https://domain-a.com",
     "https://domain-b.com/data.json",
```
[the remark this answers is not in the document]
Comment (ksuess) [reply docs/conf.py:108]: yes, indeed.

**Comment by ksuess** [comment]
Thank your for reviewing, @stevepiercy !!

**Pull request merged**